src/components/Select/Select.types.ts

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import { DropdownProps } from '../Dropdown';
+import { TextInputProps } from '../Inputs';
+import { MenuItem, MenuProps } from '../Menu';
+import { OcBaseProps } from '../OcBase';
+import { PillProps } from '../Pills';
+import { SpinnerProps } from '../Spinner';
+
+export interface SelectOption extends MenuItem {
+    selected?: boolean;
+    hideOption?: boolean;
+}
+
+export interface SelectProps extends OcBaseProps<HTMLSelectElement> {
```

Comment:
Please add size and width props to support different viewport and container sizes. See `TextInput` component for an example. They may also be passed via the `selectInputProps`, but could be obscured. May be better to surface them here and pass them down.